CONTRIBUTING.md

```diff
@@ -12,6 +12,8 @@ Here are some things you should know when contributing:
         composer install --dev
         ```
         - This could take up to about a minute depending on your internet connection.
+- We use Yarn to manage frontend dependencies.
+    - Depending on the way you installed Yarn, you will need to run either `yarnpkg` or `yarn install` to download the required dependencies.
```

Comment:
Wasn't this changed to npm?

Reply:
I never updated the documentation, oops! Not that important to update it for 2.0.3 though, CONTRIBUTING.md is not included in the release zips anyway.

Reply:
Good spot, unsure if it's worth resolving in this release at this point